### ISSUE TITLE
The training result of VGG-16 is not as good as your paper.

### ISSUE THREAD

**Issue by kouyss** [ANSSI-FR/ASCAD#5]
hi @prouff , I trained a _VGG-16_ model with the hyper-parameters you provided, batch size = 200, RMSprop optimizer with a initial learning rate of 0.00001, Desynchronization = 0, and this is already provided in your training script. 
Actually I just modified the number of _Conv1D_ of _Block12345_(Block1, 64 kernel numbers; Block2, 128 kernel numbers...) to make it fit the VGG-16 model prototype, and started the training with different epoch(75, 150, 300, 1200), but training result is not as good as your paper(page 39). 
Can you give me some advice for getting better results? thank you for your attention to this matter.

**Comment by kouyss** [comment]
![75](https://user-images.githubusercontent.com/32673788/46252034-42853f00-c495-11e8-869b-fc88ae30a474.PNG)
![150](https://user-images.githubusercontent.com/32673788/46252035-4618c600-c495-11e8-858f-3e5b5a3cf6f2.PNG)
![300](https://user-images.githubusercontent.com/32673788/46252037-47e28980-c495-11e8-992b-6c8ee0f7169a.PNG)
![1200](https://user-images.githubusercontent.com/32673788/46252038-4913b680-c495-11e8-869a-0646e4b6aed1.PNG)

**Comment by rstrullu** [comment]
Hello, did you try to make a crossvalidation for each of the number of epochs?

**Comment by prouff** [comment]
Bonjour Rémi
Merci pour ton post. Tu n'as pas parlé de learning raté ?
Amicalement

**Comment by kouyss** [comment]
I did not do this, but this does not affect the results?

**Comment by liuli9203** [comment]
How does cross-validation affect the training process ?

**Comment by prouff** [comment]
The cross-validation does not exactly affect the training process but allows for the "after the fact" validation of the average quality of the trained model.
